%YAML:1.0
image_width: 1280
image_height: 1024
camera_name: narrow_stereo
camera_matrix:
  rows: 3
  cols: 3
  data: [432.942207, 0.000000, 674.880007, 0.000000, 433.833173, 500.926404, 0.000000, 0.000000, 1.000000]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [-0.226759, 0.032128, 0.002354, -0.001643, 0.000000]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1.000000, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000, 0.000000, 0.000000, 1.000000]
projection_matrix:
  rows: 3
  cols: 4
  data: [411.200226, 0.000000, 597.498515, 0.000000, 0.000000, 446.492432, 491.548226, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000]
